metadata.yaml

```diff
@@ -20,3 +21,6 @@ requires:
     interface: k8s-service
     schema: https://raw.githubusercontent.com/canonical/operator-schemas/master/k8s-service.yaml
     versions: [v1]
+  sidebar:
+    interface: sidebar
+    optional: true
```

Comment:
are we implementing the new sidebar as well as rewriting to sidecar in this PR?  What is #38 for?

Reply:
Perfect